app/providers/[id]/page.tsx

```
@@ -0,0 +1,81 @@
+import Image from "next/image";
```

ESLint

[error] 'Image' is defined but never used.

(@typescript-eslint/no-unused-vars)